portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -8,15 +8,41 @@
 import com.google.gson.Gson;
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.datastore.*;
```

Comment:
FWIW the style guide recommends against using wildcard imports, because they can make it hard to see exactly where a class is coming from: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports